src/azure-cli/HISTORY.rst

```diff
@@ -70,6 +70,7 @@ Release History
 
 * Add new routing source type: DigitalTwinChangeEvents
 * Fix #2826: Missing features in "az iot hub create"
+* Bug Fixed: Return more descriptive message on raised exception. 
```

Comment:
Not **Bug fixed**, right? It's an enhancement. 
You could say like "{your subgroup}: Add more descriptive messages when exception raised."

Reply:
The purpose of changing from HttpOperationError to CloudError is to get more information. The catching scope of HttpOperationError is too small. Which won't give the user any informative message.